test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -184,6 +184,43 @@ def test_tf_serving_version_cpu(tensorflow_inference):
 
     stop_and_remove_container(container_name, ctx)
 
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("N/A")
+def test_tf_serving_api_version(tensorflow_inference):
+    """
+    For non-huggingface TF inference images, check that the tag version matches the version of TF serving api
+    in the container.
+
+    Huggingface includes MMS and core TF, hence the versioning scheme is based off of the underlying tensorflow
+    framework version, rather than the TF serving version.
+
+    @param tensorflow_inference: ECR image URI
+    """
+    # Set local variable to clarify contents of fixture
+    image = tensorflow_inference
+
+    if "gpu" in image:
+        cmd="pip show tensorflow-serving-api-gpu | grep Version"
+    if "cpu" in image:
+        cmd="pip show tensorflow-serving-api | grep Version"
+
```

Comment:
We really don't anticipate a new image that will miss the check above. Only thing we can think is "hpu" but that doesn't come under inference so we are good. For now I did go ahead and added an else block to handle the edge case if it may arise. Thanks for the suggestion